spec/mailers/contact_mailer_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe ContactMailer, type: :mailer do
+  describe 'mail' do
+    it 'has appropriate subject' do
+      contact = create(:contact, :with_message, copy: true)
+      mail = ContactMailer.mail(contact)
+
+      mail.should have_subject("Please confirm")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.